docs/reference/filebeat/filebeat-input-azure-blob-storage.md

```diff
@@ -122,13 +124,14 @@ $$$supported-attributes$$$
 5. [storage_url](#attrib-storage-url)
 6. [containers](#attrib-containers)
 7. [name](#attrib-container-name)
-8. [max_workers](#attrib-max_workers)
-9. [poll](#attrib-poll)
-10. [poll_interval](#attrib-poll_interval)
-11. [file_selectors](#attrib-file_selectors)
-12. [expand_event_list_from_field](#attrib-expand_event_list_from_field)
-13. [timestamp_epoch](#attrib-timestamp_epoch)
-14. [custom_properties](#attrib-custom-properties)
+8. [batch_size](#attrib-batch_size-abs)
+9. [max_workers](#attrib-max_workers)
+10. [poll](#attrib-poll)
+11. [poll_interval](#attrib-poll_interval)
+13. [file_selectors](#attrib-file_selectors)
+14. [expand_event_list_from_field](#attrib-expand_event_list_from_field)
+15. [timestamp_epoch](#attrib-timestamp_epoch)
+16. [custom_properties](#attrib-custom-properties)
```

Comment:
The list _should_ still be rendered correctly the way you have it, but:

```suggestion
12. [file_selectors](#attrib-file_selectors)
13. [expand_event_list_from_field](#attrib-expand_event_list_from_field)
14. [timestamp_epoch](#attrib-timestamp_epoch)
15. [custom_properties](#attrib-custom-properties)
```

Tip: I usually just use `1.` in front of _all_ list items in ordered lists so if I have to reorder items or add/remove items I don't have to worry about it being out of order.